src/web/components/Nav/ExpandedMenu/ExpandedMenuToggle/LabelButton.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+type Props = {
+    CHECKBOX_ID: string;
+    className: string;
+    ariaControls: string;
+    children: JSX.Element | JSX.Element[];
+};
+
+export const LabelButton = ({
+    CHECKBOX_ID,
+    className,
```

Comment:
Do we need to pass a `className` prop? I'm assuming we know what the styles are so could they just be set here in this component rather than trying to make it generic?

Reply:
No the styles are very different, I could move the styles to this component and change this to a `isVeggieBurger` boolean to conditionally render the styling?